packages/e2e/features/ui/components/authenticator/sign-up-with-username.feature

```diff
@@ -28,7 +24,6 @@ Feature: Sign Up with Username
     And I type my password
     And I confirm my password
     And I type my "email" with status "UNCONFIRMED"
-    And I type a new "preferred username"
```

Comment:
`Preferred Username` is not a sign up attribute of the `auth-with-username-no-attributes` environment.